streaming/src/main/scala/io/finch/streaming/package.scala

```diff
@@ -0,0 +1,49 @@
+package io.finch
+
+import com.twitter.io.{Buf, Reader}
+import com.twitter.util.Future
+import io.catbird.util._
+import io.finch.Endpoint.Result
+import io.finch.items.RequestItem
+import io.iteratee.{Enumeratee, Enumerator}
+
+/**
+  * Streaming module
+  */
+package object streaming {

```

Comment:
Should we extend `Module[Fturue]` (as Travis did here: https://github.com/finagle/finch/pull/557)?